3.2/Dockerfile

```diff
@@ -11,9 +11,6 @@ MAINTAINER SoftwareCollections.org <sclorg@redhat.com>
 #  * $MONGODB_DATABASE - Name of the database to create
 #  * $MONGODB_ADMIN_PASSWORD - Password of the MongoDB Admin
```

Comment:
Observation: these variables are referenced in the scripts, not in the Dockerfile, and this list is incomplete. Perhaps this information fits better in the README file? It's already there anyway.